src/main/java/sdmExam/Board.java

```diff
@@ -76,7 +76,88 @@ public boolean existsDiagonallyAdjacentWithStone(Intersection intersection, Ston
                 .anyMatch(diagonalIntersection -> diagonalIntersection.hasStone(stone));
     }
 
+    public ArrayList<ArrayList<Intersection>> findTerritories() {
+        ArrayList<ArrayList<Intersection>> territories = new ArrayList<>();
+        ArrayList<Intersection> territory = new ArrayList<>();
+
+        for(int col = 1; col <= Board.BOARD_SIZE; col++){
+            Intersection intersection = new Intersection(Position.in(3, col), Stone.NONE);
```

Comment:
Why do we have 3 as hard coded here? (In Position.in(3, col)). Are we looking for territories only in row 3?